src/functionalTest/resources/application-functional.yaml

```diff
@@ -15,7 +15,7 @@ wa-post-deployment-test:
 azure:
   servicebus:
     connection-string: ${AZURE_SERVICE_BUS_CONNECTION_STRING}
-    topic: ${AZURE_SERVICE_BUS_TOPIC_NAME:ccd-case-events-aat}
+    topic: st-ccd-case-event
```

Comment:
is this the correct topic name? where did you get the value from?

Reply:
It was what was set before in the build.gradle file. I tried to use what had been used in the PR below but this was the only value that got the functional tests to run.

https://github.com/hmcts/sptribs-case-api/pull/1773